force-app/main/default/lwc/miscPermissionBasedUI/miscPermissionBasedUI.js-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>49.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+        <target>lightning__AppPage</target>
```

Comment:
Across all recipes we set all three targets: App, Home, and Record.

Reply:
I fixed this. But I'd like to ask: why?

Reply:
Simple showcase on how to set multiple targets. Also enables the learner to easily use/play with any of the components on any of the most relevant LEX experiences.